networkx/algorithms/flow/tests/test_networksimplex.py

```diff
@@ -6,6 +6,7 @@
 import pytest
 
 import networkx as nx
+from networkx import NetworkXUnbounded

```

Comment:
Just a style note - it's preferred to refer to exceptions from the main namespace to keep the imports down (see corresponding change below)

```suggestion

```